src/store/reducers.js

```diff
@@ -1,10 +1,11 @@
 import app from './app/reducer';
 import auth from './auth/reducer';
 import user from './user/reducer';
+import { usersReducer as users } from './users/reducer';
```

Comment:
I guess, instead of creating **users**, we can use them inside **user**. But I'm not sure.

If you decide to keep this way, can you `export default`?

Reply:
@edencakir `users` inside `user` object doesn't sound intuitive & will create more confusions.